.circleci/config.yml

```diff
@@ -133,9 +141,37 @@ jobs:
       - run: yarn install
       - run: ./scripts/promote
 
+  run-all-nuts:
+    docker:
+      - image: node:latest
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v3-npm-{{checksum "yarn.lock"}}
+      - run: yarn install
+      - run: ./scripts/run-plugin-nuts.js
+
+  monitor-all-nuts:
```

Comment:
Job that takes in pipeline instance data and then monitors each one until completion.